.changelog/39477655a9814e22ad526d7755c34560.json

```diff
@@ -0,0 +1,11 @@
+{
+    "id": "39477655-a981-4e22-ad52-6d7755c34560",
+    "type": "feature",
+    "description": "New API client",
+    "modules": [
+        ".",
```

Comment:
Looks like you are include the root module with this change note.

Reply:
This note can be deleted all together since this is auto created as a release note.